website/docs/developer/development-guide/code/versioning-releases.md

```diff
@@ -84,11 +82,13 @@ the release process is complete.
 
 **Merge the base branch back into develop and master**
 
-Finally, you need to merge the version update changes back into develop and 
-master.
+If this is the latest version, you need to merge the version update changes back into develop, then synchronize develop and master.
 
-If this is the latest version, create the PRs to merge the release branch 
-into develop and master. 
+1. Create a PR to merge the release branch into develop.
+   This should only contain changes from the release process.
+1. Once complete, create a PR to merge develop into master.

```

Comment:
This could be problematic and I would not recommend merging develop into master. Unless you plan on freezing any PR merges into develop when you are working on a release. This could cause untested and unplanned changes being released into master.

To avoid this, I would only merge the release branch into master.

Reply:
True. Updated.